### DISCUSSION TITLE
MathJax in List that Includes Both Block and Inline Issue

### DISCUSSION THREAD

**Discussion by stevesimoniintel** [community/community#40775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There is an issue when MathJax is embedded in a math block as well as inline in a markdown list element as rendered by GitHub

The 
[GitHub MathJax Block and Inline in List Element Gist](https://gist.github.com/stevesimoniintel/c872baa32c517a735c4e8a97e1e31745)
illustrates the issue.

The
[GitHub MathJax Block in List Element](https://gist.github.com/stevesimoniintel/89075513ccd0ec8e7ae0f942eab5b979)
illustrates that the MathJax can be included as a block without issue.

It could be that I'm making some sort of mistake but I believe that the syntax is correct.
Thanks

**Comment by ossanna16** [comment]
Hi there @stevesimoniintel and welcome to our community! Thank you for reporting this!

**Comment by nschloe** [comment]
May be a duplicate of https://github.com/community/community/discussions/17325.

_Edit:_ Added this to my math bug overview. https://github.com/nschloe/github-math-bugs
